example/ios/Flutter/AppFrameworkInfo.plist

```diff
@@ -21,6 +21,6 @@
   <key>CFBundleVersion</key>
   <string>1.0</string>
   <key>MinimumOSVersion</key>
-  <string>11.0</string>
+  <string>12.0</string>
```

Comment:
Let's hold off these changes until we drop support for flutter versions < 3. Which should be in the near future.